sky/packages/sky/lib/src/widgets/gesture_detector.dart

```diff
@@ -27,7 +28,10 @@ class GestureDetector extends StatefulComponent {
     this.onHorizontalDragEnd,
     this.onPanStart,
     this.onPanUpdate,
-    this.onPanEnd
+    this.onPanEnd,
+    this.onScaleStart,
+    this.onScaleUpdate,
+    this.onScaleEnd
   }) : super(key: key);
```

Comment:
Did you want to add an assert about not using both pan and scale gestures at the same time?